### ISSUE TITLE
No license

### ISSUE THREAD

**Issue by palfrey** [JulianBirch/Hendrix#1]
This project doesn't appear to have a license specified. Which one is it released under?

**Comment by JulianBirch** [comment]
You're right, I should fix the project.clj. It should read EPL, like most Clojure projects.
